src/transformers/models/whisper/modeling_flax_whisper.py

```diff
@@ -58,6 +60,16 @@
 remat = nn_partitioning.remat
 
 
+# Copied from transformers.models.whisper.modeling_whisper.sinusoids
+def sinusoids(length: int, channels: int, max_timescale: float = 10000) -> np.ndarray:
```

Comment:
Sure, that works as well. I used numpy initially to generate the sinusoids so that we can copy it across the 3 files and avoid errors. But having separate functions is fine by me too.